docs/sdk-and-tools/indices/rounds.md

```diff
@@ -0,0 +1,42 @@
+---
+id: es-index-rounds
+title: Elasticsearch index: rounds
+---
+
+The `_id` field of this index is composed in this way: `{shardID}_{round}` (example: `2_10905514`)
+
+
+| Field            | Description                                                                                                            |
+|------------------|------------------------------------------------------------------------------------------------------------------------|
+| round            | The round field represents the number of the round.                                                                    |
+| signersIndexes   | The signersIndexes is an array that contains the indices of the validators that should sign the block from this round. |
+| blockWasProposed | The blockWasProposed is true if a block was produced in this round.                                                    |
```

Comment:
[comment]

Flag name is `Proposed`, while comment says `produced`

Reply:
changed.